### PR TITLE
Added test cases for getTitle,setTitle,getImage,setImage

### PR DESCRIPTION
**Pull request by kalyaniIN**

### CODE REVIEW

File: src/test/java/com/example/produktapi/model/ProductTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package com.example.produktapi.model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+class ProductTest {
+    // Create a Product instance
```
Comment (Uma-Gopalakrishnan) [inline src/test/java/com/example/produktapi/model/ProductTest.java:7]: Pls remove this code

**Review by Uma-Gopalakrishnan — CHANGES_REQUESTED** [review]
Product instance can be reused from master

**Comment by kalyaniIN** [comment]
I have updated the branch

**Review by Uma-Gopalakrishnan — APPROVED** [review]
Approved

**Pull request merged**